en/components/date-picker.md

```diff
@@ -50,7 +50,7 @@ Akin to the Calendar, the Date Picker lets you pick between three selection mode
 
 ## Buttons
 
-The Drop-down Date Picker comes with two buttons: one for closing in and keeping the current selection, and one for navigating to today. The Button area is templatable allowing you, for example, to hide a button by setting its override to ~No Symbol, or assign a different action to it. The Dialog Date Picker comes with no buttons by default because upon selecting a date, the dialog closes automatically.
+The Drop-down Date Picker comes with two buttons: one for closing in and keeping the current selection, and one for navigating to today. The Button area is templatable, allowing you, for example, to hide a button. To hide a button in Sketch, you have to set its override to ~No Symbol, and through the use of a smart layout, it will adjust accordingly. While in Adobe XD, you have to delete the unused layer instead, and the layout will adjust through the use of stacks. The Dialog Date Picker comes with no buttons by default because upon selecting a date, the dialog closes automatically.
```

Comment:
Buttons are not available anymore for date-picker, they exist by default only for the time-picker. Remove this section altogether from this topic, please.